src/modules/appUI.js

```diff
@@ -12,16 +36,31 @@ const filmTemplate = (info) => `
           <i class="far fa-heart"></i>
         </div>
       </div>
-      <button type="button" class="comments">Comments</button>
+      <button type="button" class="comments" id="${info.id}">Comments</button>
     </li>
   `;
-
-const displayMovies = () => {
+const displayMovies = async () => {
   const filmsContainer = document.querySelector('.films');
+
```

Comment:
I suggest we also remove blank lines here and only use them to separate large and logical code blocks. This helps to make our code base look professional and clean.

Reply:
This is your code tho.

Reply:
I'm referring to the blank line and not the code on line 43. Please check the comment again. 😃